css/general.css

```diff
@@ -0,0 +1,63 @@
+* {
+	font-family: 'Poppins', sans-serif;
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	outline: none;
+	border: none;
+	text-decoration: none;
+	text-transform: capitalize;
+	transition: all .2s linear;
+}
+
+html {
+	font-size: 62.5%;
+	overflow-x: hidden;
+	scroll-padding-top: 7rem;
+	scroll-behavior: smooth;
+}
+
+html::-webkit-scrollbar {
+	width: .8rem;
+}
+
+html::-webkit-scrollbar-track {
+	background: transparent;
+}
+
+html::-webkit-scrollbar-thumb {
+	background: var(--primary);
+	border-radius: 5rem;
+}
+
+
+
+section {
+	padding: 2rem 7%;
+}
+
+.heading {
+	text-align: center;
+	padding-bottom: 2rem;
+	color: var(--text-color-1);
+	font-size: 4rem;
+}
+
+.heading span {
+	position: relative;
+	z-index: 0;
+}s
```

Comment:
I guess this is by mistake

Reply:
yes!  i was trying to implement some styles but forgot to remove them.